examples/multimodule-projects-on-kubernetes-example/project-tests-on-kubernetes-example/src/test/java/io/dekorate/example/SpringBootForMultiModuleAppsOnKubernetesIT.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2021 The original authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.dekorate.example;
+
+import io.dekorate.testing.annotation.KubernetesIntegrationTest;
+import io.dekorate.testing.annotation.Named;
+import io.fabric8.kubernetes.api.model.KubernetesList;
+import io.fabric8.kubernetes.api.model.Pod;
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.kubernetes.client.LocalPortForward;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import io.dekorate.testing.annotation.Inject;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.net.URL;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@KubernetesIntegrationTest
+class SpringBootForMultiModuleAppsOnKubernetesIT {
+
+  @Inject
+  private KubernetesClient client;
+
+  @Inject
+  @Named("project-a-on-kubernetes-example")
+  private KubernetesList resourcesForProjectA;
+
+  @Inject
+  @Named("multimodule-project-a-on-kubernetes-example")
```

Comment:
The first is the name of the service/pod, the second is the name of the project file.

Reply:
I think that it would be better if each qualifier kept to a single and specific use (e.g. @Named only used for resource name e.g. Service, Pod etc). So, I am wondering if we should use another qualifier for the `project file`. However, it is still not 100% clear to me how the multimodule thing works so, I don't have something more concrete to say.

Reply:
Good point @iocanel 
So, let's aggregate all the Kubernetes items loaded by each project into a single KubernetesList, so we won't need the `@Named` annotation at all.